.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,16 +16,16 @@
 -->
 
 [ ] Documentation update
-[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
-[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
-[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
+[ ] Bug fix ([template](ISSUE_TEMPLATE/bug-report.yml))
+[ ] New rule ([template](ISSUE_TEMPLATE/new-rule.yml))
+[ ] Changes an existing rule ([template](ISSUE_TEMPLATE/rule-change.yml))
```

Comment:
To be honest, I would prefer from a maintainer standpoint that contributors just open an issue corresponding to the template.  That way they can provide the information you guys need **without having to maintain the same templates twice on your end**.  As for copy/paste, contributors can still copy/paste from GitHub's rendered version of the YML.

Reply:
I feel like copy pasting the markdown is better than the rendered YML.

> without having to maintain the same templates twice (in YML and MD format) on your end

Also these templates are not changed regularly so it would be better if leave them as suc?

Reply:
Perhaps not, but there is already divergence between these two template file types, which feels problematic already. Separately, I don't see why users shouldn't just file an issue in the first place.

Reply:
>Separately, I don't entirely see why users can't/shouldn't just file an issue in the first place.

Sometimes the fix is a one-liner or very simple to implement that contributors directly submit a PR, in such cases this is helpful.

Reply:
A simple fix does not necessarily mean the problem itself is easy to communicate to maintainers.

IMO in most cases I as a maintainer would prefer to see triage separate from a PR.  Consider the templates that there: should contributors really be opening PRs for new rules or rule changes without opening an issue first?  More often than not, I think there turns out to be more to discuss that would be better suited in an issue rather than a PR.

I'm still not seeing the benefit of maintaining these templates in two different forms when the YML versions appear to be more developed content-wise than the Markdown ones.  If it is truly the case that an issue is trivial to communicate and fix, I don't think you really need a template, let alone this many.